src/backend/libpq/rangerrest.c

```diff
@@ -68,51 +69,99 @@ static void getClientIP(char *remote_host)
 	}
 }
 
-RangerACLResult parse_ranger_response(char* buffer)
+/*
+ * parse ranger response
+ * @param	buffer	ranger response 	
+ * @param	result_list		List of RangerPrivilegeResults
+ * @return	0 parse success; -1 other error
+ */
+static int parse_ranger_response(char* buffer, List *result_list)
 {
 	if (buffer == NULL || strlen(buffer) == 0)
-		return RANGERCHECK_UNKNOWN;
+		return -1;
```

Comment:
I modified this function's return value.
The acl check results will be stored in the param `result_list`.